frontend/src/views/BoardGame.tsx

```diff
@@ -41,6 +42,14 @@ export const BoardGame: FC<BoardGameProps> = (props: BoardGameProps) => {
       setCurrentUsers(currentStatus.users);
     }
   }, [lastMessage]);
+  
+  useEffect(() => {
+    window.onbeforeunload = function () {
+      if (currentUsers.length || cards.length) {
+        return "Are you sure you want to navigate away?";
```

Comment:
What about something about leaving the empathy room? "Are you sure you wish to leave this Empathy Room?"